yt/frontends/stream/data_structures.py

```diff
@@ -1101,15 +1100,18 @@ def load_particles(data, length_unit = None, bbox=None,
     r"""Load a set of particles into yt as a
     :class:`~yt.frontends.stream.data_structures.StreamParticleHandler`.
 
-    This should allow a collection of particle data to be loaded directly into
+    This will allow a collection of particle data to be loaded directly into
     yt and analyzed as would any others.  This comes with several caveats:
 
-    * There must be sufficient space in memory to contain both the particle
-      data and the octree used to index the particles.
+    * There must be sufficient space in memory to contain all the particle
+      data.
     * Parallelism will be disappointing or non-existent in most cases.
+    * Fluid fields are not supported.
 
-    This will initialize an Octree of data.  Note that fluid fields will not
-    work yet, or possibly ever.
+    Note: in order for the dataset to take advantage of SPH functionality,
```

Comment:
I'm confused.  The sample dataset FIRE_M12I_ref11 does not contain these two fields, and yet it appears to work with SPH functionality.  Am I missing something here?  Must they be in `io` field type?  Because that dataset has both `('gas', 'density')` and `('gas', 'smoothing_length')`.

Reply:
This is specifically for data loaded via `yt.load_particles`

Reply:
Oh! Now I see this is contained in the docstring for that method.  My bad.  LGTM.